### ISSUE TITLE
Prefer Players on the Closest Y coord (aka people on the same level)

### ISSUE THREAD

**Issue by TeamIlluminati** [chriskvm/ShellshockIoAimbot#16]

**Comment by TeamIlluminati** [comment]
A suggested enhancement would be to prefer players on the closest Y coordinate, aka people on the same level. This would minimize the looking down at the floor or ceiling a lot.  Another suggestion would be to prefer players recently attacked (e.g. so if you are finishing someone off, it prefers that player instead of suddenly turning around to shoot someone through a wall). Perhaps there is a way to hook the map coords for structures and map out where all the objects are and make it so  you don't aim through walls (unless you want to, it can be used to follow people quite well).  Lastly, I'm sure there is a way to find health for all the players, I can add that into the box esp I'm making (see my fork) and you/we could most likely incorporate a "smart aim" that aims at the closest person w/ lowest health not through a wall. Perhaps also make these parameters adjustable/changeable?

**Comment by chriskvm** [comment]
Also in-engine ESP is easy enough to do like this:
```
nameSprites.renderingGroupId = 2;
```

**Comment by TeamIlluminati** [comment]
That's true, probably easier too. I suppose we could also have it render outlines/shades for enemies like how it does for your own teammates. That way, it would be very easy to implement.
